polaris-pipeline/Common/Dto/Response/Case/DefendantsAndChargesListDto.cs

```diff
@@ -1,29 +1,18 @@
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
-using Common.Dto.Response.Document.FeatureFlags;
 using Newtonsoft.Json;
 
 namespace Common.Dto.Response.Case
 {
-    public class DefendantsAndChargesListDto
+    public class DefendantsAndChargesListDto : DefendantsAndChargesListCoreDto
     {
         public DefendantsAndChargesListDto()
-        {
-
-        }
-
-        [JsonProperty("caseId")]
-        [JsonPropertyName("caseId")]
-        public int CaseId { get; set; }
-
-        [JsonProperty("versionId")]
-        [JsonPropertyName("versionId")]
-        public long VersionId { get; set; }
+        { }
 
         [JsonProperty("defendants")]
         [JsonPropertyName("defendants")]
-        public IEnumerable<DefendantAndChargesDto> DefendantsAndCharges { get; set; } = new List<DefendantAndChargesDto>();
+        public IList<DefendantAndChargesDto> DefendantsAndCharges { get; set; } = new List<DefendantAndChargesDto>();
 
-        public PresentationFlagsDto PresentationFlags { get; set; } = new PresentationFlagsDto();
+        public override int DefendantCount => DefendantsAndCharges?.Count ?? 0;
```

Comment:
The minimal information the (current) orchestration logic needs is whether a case has multiple defendants or not.